### PR TITLE
Write Debriefing

### LINKED ISSUE #7: Write debriefing
- [x] Main design challenge
- [x] Client description
- [x] Description of assignment
- [x] Stakeholder definition (target audience, client, CMD, us)
- [x] Plan of action
  - [x] Research requirements(???)
  - [x] Project planning
  - [x] Task delegation
  - [x] Deliverables

### PR DESCRIPTION
**Pull request by jonahgoldwastaken**
Closes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,147 +1,202 @@
...
-3. Woensdag is er ruimte voor workshops en code/design reviews.
-4. Donderdag is er ruimte voor workshops en code/design reviews.
-5. Vrijdag presenteer je je 1e iteratie van het prototype aan de opdrachtgever.
+For the client it is important that she has a clear idea of our progress during this project. To follow the progress of this project it is important to the client that we submit a prototype at the end of every week and discuss the current progress.
 
-### Meesterproef - Week 2 (24 - 28 mei)
+### CMD
 
-De derde week ga je weer testen bij de opdrachtgever. Aan het eind van de week laat je de 2e iteratie van het protoype zien.
+This project has been made available to us by the the course Communication & Multimediadesign of the Hogeschool van Amsterdam. For this project we are assigned a coach (Vasilis van Gemert) that will coach us during the process of this project. Every week we will be meeting with our coach to discuss the current progress of the project and receive feedback and guidance. To be graded at the end of the project we will have to provide the following deliverables:
```
Comment (benl95) [inline README.md]: Change sentence "assigned a coach" to "assigned to a coach".

File: README.md [hunk 2]
```diff
@@ -1,147 +1,202 @@
...
+Based on these 4 deliverables we will be graded.
 
-De derde week ga je weer testen bij de opdrachtgever. Aan het eind van de week laat je de 3e iteratie van het protoype zien.
+### Target audience
 
-1. Maandag heb je een standup en checkup met je coach
-2. Woensdag is er ruimte voor workshops en code/design reviews.
-3. Donderdag is er ruimte voor workshops en code/design reviews.
-4. Vrijdag presenteer je je 3e iteratie van het prototype aan de opdrachtgever.
+The target audience consists of two groups. The functional illiterate minorities and the volunteers that will help them explaining the letters. It is important to the target audience that an application is provided that fullfills their needs. For the functional illiterate minorities it is important that the application has the following aspects:
```
Comment (benl95) [inline README.md]: Change "functional illiterate minorites" to "functional illiterate immigrants/people".

File: README.md [hunk 3]
```diff
@@ -1,147 +1,202 @@
...
+- The application should be able to provide explanation in the mother tongue of the user
 
-Deze week ga je weer testen bij de opdrachtgever. Aan het eind van de week laat je de 4e iteratie van het protoype zien.
+For the volunteers it is important that the application provides the following aspects:
 
-1. Maandag heb je een standup en checkup met je coach.
-2. Woensdag is er ruimte voor workshops en code/design reviews.
-3. Donderdag is er ruimte voor workshops en code/design reviews.
-4. Vrijdag presenteer je je 4e iteratie van het prototype aan de opdrachtgever.
+- Volunteers need to able provide a spoken explanation for the letters
```
Comment (benl95) [inline README.md]: Change "need to able" to "need to be able to"

**Review by benl95 — CHANGES_REQUESTED** [review]
Placed comments for spelling/language errors

File: README.md (hunk 2 above)
Comment (jonahgoldwastaken) [reply README.md]: Changed it to "functional illiterate people"

**Comment by jonahgoldwastaken** [comment]
@benl95 typos fixed :)

**Review by benl95 — APPROVED** [review]
Approved

**Pull request merged**